content/en/role-based-access-control.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Role-Based Access Control (RBAC)
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+
+Role-Based Access Control (RBAC) is a method to regulate system and resource access based on individual users' roles within an organization.
+Each job function within an organization has a different role with specific permissions. 
+For instance, a marketing team member may have permission to view the deals sales has in the pipeline (*read* permission) but won't be allowed to edit them (*write* permission). 
+Sales and marketing have different permissions based on their roles, hence role-based access control. 
+
+## Problem it addresses
+
+Without RBAC, it is easy to lose track of who has access to what. Malicious actors are always in search of vulnerabilities to exploit and a mismanaged account with too many privileges can put an entire system at risk. 
```

Comment:
I agree with @CathPag and @iamNoah1 
I think we'd better focus on `role` and not `access control` in the problem it address.

following is my suggestion. (please improve it if necessary)

```suggestion
Individually managing access permissions to many users in an organization of various system resources and data is very complex considering that each user will have different set of resources the user can access. For instance, if there are 100,000 users and 1,000 resources in an organization, an administrator needs to check whether each user has authority and responsibility for each resource 100,000 times.
Therefore, it is necessary to simplify the access control according to role groups based on authority and responsibility.
```